src/main/kotlin/frc/robot/robotstate/RobotStates.kt

```diff
@@ -0,0 +1,8 @@
+package frc.robot.robotstate
+
+enum class ROBOT_STATE() {
+    Idling,
+    Intaking,
+    Shooting
+}
```

Comment:
The constants can be lowercase:
> For enum constants, it's OK to use either all uppercase, underscore-separated ([screaming snake case](https://en.wikipedia.org/wiki/Snake_case)) names (`enum class Color { RED, GREEN }`) or upper camel case names, depending on the usage.

But the class name is just a regular class name. Here is another example from the official Kotlin docs:
```kt
enum class Direction {
    NORTH,
    SOUTH,
    WEST,
    EAST, // trailing comma
}
```
 I'd do the following:
```suggestion
enum class RobotState() {
    IDLE,
    INTAKING,
    SHOOTING
}
```
But this is also fine:
```suggestion
enum class RobotState() {
    Idle,
    Intaking,
    Shooting
}
```
Note that it's "intaking", not "intakeing".